src/app/sl-public/components/sl-import/sl-import.component.html

```diff
@@ -0,0 +1,110 @@
+<div class="jumbotron jumbotron-fluid">
+  <div class="container">
+    <h1 class="display-3">Import Sanctionslist</h1>
+    <br />
+    <p class="lead">On this page you can Import a new sanctionlist. But be aware the file
+      <b>must be</b> a
+      <b>.xml</b> file. You can get the sanctionlist from the
+      <a href="https://data.europa.eu/euodp/en/data/dataset/consolidated-list-of-persons-groups-and-entities-subject-to-eu-financial-sanctions"
+        target="_blank">EU</a> or
+      <a href="https://www.seco.admin.ch/seco/de/home/Aussenwirtschaftspolitik_Wirtschaftliche_Zusammenarbeit/Wirtschaftsbeziehungen/exportkontrollen-und-sanktionen/sanktionen-embargos.html"
+        target="_blank">SECO</a> website.</p>
+  </div>
+</div>
+<div class="container">
+  <label class="btn btn-primary btn-s">
+    <span class="fa fa-search"></span>
+    Browse&hellip;
+    <input type="file" accept=".xml" ng2FileSelect [uploader]="uploader" style="display: none;" multiple />
+  </label>
+  <br />
+  <br />
+  <div class="container">
+    <h3>Upload queue</h3>
+    <p>Queue length: {{uploader?.queue?.length}}</p>
+    <table class="table">
+      <thead>
+        <tr>
+          <th width="50%">Name</th>
+          <th>Size</th>
+          <th>Progress</th>
+          <th>Status</th>
+          <th>Source</th>
+          <th>Actions</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr *ngFor="let item of uploader.queue">
+          <td valign="middle">
+            <strong *ngIf="checkFileExt(item.file.type);" class="error-import-info">{{item?.file?.name}}</strong>
```

Comment:
Maybe here is some a more beautiful solution possible... but I didn't find one...

Reply:
validate the files in the controller or use directives. but I think it's ok at the moment.